src/wizard/Wizard.jsx

```diff
@@ -213,7 +222,7 @@ class Wizard extends React.Component {
                                 {step.label}
                             </StepButton>
 
-                            {step.help && step === currentStep ? <Help step={step} /> : null}
```

Comment:
Side-effect on metadata-sync. Now help button is always rendered even when ```step.help``` is undefined.

Proposal: ``` {step.help && <Help step={step} currentStepKey={currentStepKey} /> }```

Reply:
> Help button is always rendered even when step.help is undefined

Indeed! Your proposal looks fine to me.